node/src/Commands.ts

```diff
@@ -2293,15 +2293,15 @@ export function createFunctionLoad(
 /** Optional arguments for `FUNCTION LIST` command. */
 export type FunctionListOptions = {
     /** A wildcard pattern for matching library names. */
-    libNamePattern?: string;
+    libNamePattern?: GlideString;
     /** Specifies whether to request the library code from the server or not. */
     withCode?: boolean;
 };
 
 /** Type of the response of `FUNCTION LIST` command. */
 export type FunctionListResponse = Record<
     string,
-    string | Record<string, string | string[]>[]
+    GlideString | Record<string, GlideString | GlideString[]>[]
```

Comment:
For `Record< , >`, the first argument cannot be changed from `string` to `GlideString`. Is this the right change?

Reply:
Could we use `Map` instead for this case?

Reply:
It is done in the rust part of the client actually

Reply:
https://github.com/valkey-io/valkey-glide/issues/2152

Reply:
As per today discussion, I will change it to array.

Reply:
I may need to consult with Yury on making these changes since it will cause breakage on utilities' functions.